string msgDescriptor
string objectDescriptor
string objectID
float64 longitude
float64 latitude
float64 heading
